src/App.vue

```diff
@@ -97,11 +97,13 @@ export default {
   mounted() {
     // TODO: REMOVE, call action directly
     this.$bus.$on('SAVE_NOTES', this.saveNotes);
```

Comment:
if you don't mind, can you add the same comment here?

`// TODO: REMOVE, call action directly`

From time to time I go around and tackle all the `TODOs`